_posts/2019-10-09-hhvm-4.26.0.markdown

```diff
@@ -0,0 +1,44 @@
+---
+title: "HHVM 4.26.0"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.26 is released! This release marks the end of support for 4.20;
+4.21&ndash;4.25 remain supported, as do the LTS releases 3.30 and 4.8.
+
+# Highlights
+
+- various IDE integration improvements (better autocompletion, more contextual information on hover)
+- values with [`dynamic` type](https://docs.hhvm.com/hack/built-in-types/dynamic) can now be used in more places (inside XHP, `unset` statements)
+- it is no longer a Hack error to declare a method with the same name as its encompassing class (previously error 3007)
+- closing an SQLite connection when there are unfinalized prepared statements will now schedule the connection to be [closed later](https://www.sqlite.org/c3ref/close.html), instead of returning an error
+- an experimental new syntax was added for union types `(Cat | Dog)` and intersection types `(FourLegged & Mammal)` &ndash; this is an early experimental prototype not meant for general use, but if you want to experiment with it, add `union_intersection_type_hints=true` to your `.hhconfig`
```

Comment:
@andrewjkennedy @dlreeves Is this a reasonable thing to say?

I can leave this out of the release notes if you prefer but in my experience it's probably better to mention things and make it clear they're experimental, than to not mention it and hope no one notices and starts using it :)